webpack.config.js

```diff
@@ -115,6 +119,7 @@ const federationConfig = {
         "./DrugChartModal": "./src/entries/DrugChartModal/DrugChartModal.jsx",
         "./DrugChartModalNotification":
           "./src/entries/DrugChartModal/DrugChartModalNotification.jsx",
+        "./IpdDashboard": "./src/components/Dashboard/Dashboard.jsx",
```

Comment:
Dashboard is a container for other display controls and is an entry point, it should sit inside entries folder